### PR TITLE
TFW-3122 - fix to US page scroll

### PR DESCRIPTION
**Pull request by jasonhale**
Introduced a vertical scrollbar placement problem with TFW-2941.  This fixes that issue.

Screenshot of initial prob:
Scrollbar on US

### CODE REVIEW

File: assets/sass/base/_mixins.scss [hunk 1]
```diff
@@ -64,7 +64,7 @@
     position: relative;
     width: 100%;
     max-width: $max-content-width;
-    height: 100%;
```
Comment (troyblank) [inline assets/sass/base/_mixins.scss]: This causes the issue that the welcome graphic won't sit at the bottom of the screen.

Screen Shot 2019-03-11 at 12 28 15 PM

Comment (jasonhale) [reply assets/sass/base/_mixins.scss]: Awesome, thanks

**Comment by jasonhale** [comment]
@troyblank - ok, updated.  Thanks for catching.  I reverted the style changes introduced into the splash page component (also, it would've modified the splash as it is on Workflow), and then added a container in US to get it layed out correctly.

**Comment by troyblank** [comment]
@jasonhale sorry man just one more small area it's not flush to the bottom.

Screen Shot 2019-03-11 at 2 21 23 PM

**Comment by jasonhale** [comment]
No prob.  Updated.

**Pull request merged**